spec/fixtures/sshd_config_debian

```diff
@@ -99,6 +99,18 @@ UsePAM yes
 # Accept locale-related environment variables
 AcceptEnv LANG
 AcceptEnv LC_CTYPE
+AcceptEnv LC_NUMERIC
```

Comment:
you shouldn't be adding to these fixtures files. These are fixtures of what works for those OS's and should not change with your PR.